packages/wonder-blocks-dropdown/package.json

```diff
@@ -14,6 +14,7 @@
     "@khanacademy/wonder-blocks-button": "0.0.3",
     "@khanacademy/wonder-blocks-color": "^1.0.0",
     "@khanacademy/wonder-blocks-core": "^1.0.0",
+    "@khanacademy/wonder-blocks-icon": "0.0.1",
```

Comment:
(I think this will need to get updated - and we should un-pin it so that it updates in the future!)